_data/2020_checker.yml

```diff
@@ -48,6 +53,10 @@ en:
                 <li>Administrators - please see <a href="#action-authoritative-dns-operators">description for DNS admins</a>
                         and correct problems reported in the technical report below.</li>
             </ul>
+    clientReportFail:
+        short: "Fatal errors detected!"
+        long: >
+            todo
```

Comment:
I wonder how to make this informative. Unfortunatelly we cannot simply point to ISP because the client machine can use different DNS resolver ...

Do you think it makes sense to link to http://www.whoismydns.com/ or similar service?

Reply:
I donno, that's why I asked for help.

There can be a failing resolver in the middle also, a lot of ISP's forwards to public DNS for example.

Reply:
Suggested text:

> The test could not be completed due to a network problem. This problem could be on your local machine, in your local network, or with whomever runs the DNS servers you are using. This website cannot determine where the problem is happening. You can use <a href="Who is My DNS">http://whoismydns.com/</a> or the 'Network' button in <a href="Check My DNS">https://cmdns.dev.dns-oarc.net/</a> to find out who runs your DNS servers.

The problem is that neither of those sites actually tell me who my ISP is. Who is My DNS only does ARIN lookups, and cmdns shows netnames, which often do not mention the actual ISP. Mentioning org-name, or finding the ASN owner, would be more useful.